docs/scaffold.md

```diff
@@ -0,0 +1,28 @@
+# Start project
+
+## Overview
+
+To see how easy is using Bigflow you can use `start-project` which creates example standalone project with [standard structure](https://github.com/allegro/bigflow/blob/master/docs/project_setup_and_build.md#project-structure).
+This project contains Apache Beam workflow and BigQuery workflow.
+You will be able to deploy those workflows in your GCP project providing informations like project_id/bucket etc.
+
+Beam workflow example counts letters in array and then saves this information to file in your bucket.
+BigQuery workflow example creates necessary tables then populates one of them and finally moves data from one table to another.
+
+
+## Start project
+To start a new project type in terminal 
+
+    bigflow start-project 
+
+## Preparation
+  1. Log in to GCP using Google Cloud SDK:
+  
+    gcloud auth application-default login.
+    
+  1. Create a [Cloud Composer](https://cloud.google.com/composer/docs/how-to/managing/creating#creating_a_new_environment) instance.
+  1. You need to create GCS bucket with same name as your project and inside this bucket create directories `beam_runner/staging` and `beam_runner/temp`. This bucket is for dataflow processing.
```

Comment:
1. You need to create a GCS bucket with the same name as your project and inside this bucket create directories `beam_runner/staging` and `beam_runner/temp`. 

This bucket is for dataflow processing. <- to ostatnie zdanie brzmi tak średnio, ale nie wiem jak zmienić :(